test/constants.ts

```diff
@@ -0,0 +1,34 @@
+import { PR } from "src/types"
+
+export const org = "org"
+export const repo = "repo"
+export const user = "user"
+
+export const team = "team"
+export const userCoworker = "userCoworker"
+export const userCoworker2 = "userCoworker2"
+export const userCoworker3 = "userCoworker3"
+export const coworkers = [userCoworker, userCoworker2]
+
+export const configFilePath = "pr-custom-review.yml"
+export const prNumber = 1
+export const condition = "condition"
+
+export const githubApi = "https://api.github.com"
+export const githubWebsite = "https://github.com"
+export const reviewsApiPath = `/repos/${org}/${repo}/pulls/${prNumber}/reviews`
```

Comment:
I think these constants might be useful for the app itself. Not only for tests

Reply:
We're not using any GitHub API endpoint directly in the application code because the Octokit library already constructs the right URL based on the parameters, e.g.

`octokit.rest.pulls.listReviews({ owner, repo, pull_number })`

So no, they are not useful outside of tests